monitoring/definitions/otel_collector.go

```diff
@@ -0,0 +1,169 @@
+package definitions
+
+import (
+	"github.com/sourcegraph/sourcegraph/monitoring/definitions/shared"
+	"github.com/sourcegraph/sourcegraph/monitoring/monitoring"
+	"time"
+)
+
+func OtelCollector() *monitoring.Dashboard {
+	containerName := "otel-collector"
+
+	return &monitoring.Dashboard{
+		Name:        containerName,
+		Title:       "OpenTelemetry Collector",
+		Description: "The OpenTelemetry collector ingests OpenTelemetry data from Sourcegraph and exports it to the configured backends.",
+		Groups: []monitoring.Group{
+			{
+				Title:  "Receivers",
+				Hidden: false,
+				Rows: []monitoring.Row{
+					{
+						{
+							Name:        "otel_span_receive_rate",
+							Description: "spans received per receiver per second",
```

Comment:
sorry, second update - you aggregate on minute, so we should say minute here:

```suggestion
							Description: "spans received per receiver per minute",
```